mods/d2k/rules/infantry.yaml

```diff
@@ -49,6 +42,7 @@ engineer:
 	Captures:
 		CaptureTypes: husk
 	-AutoTarget:
+	-TakeCover:
```

Comment:
The AssetBrowser suggests otherwise, if I'm not mistaking. Likely the sequence just isn't defined, in which case better leave a note here saying that the engineer is missing his sequence.